cmd/uplink/cmd/ls.go

```diff
@@ -93,7 +93,7 @@ func listFiles(ctx context.Context, bs buckets.Store, u *url.URL, prependBucket
 	startAfter := paths.New("")
 
 	for {
-		items, more, err := o.List(ctx, paths.New(u.Path), startAfter, nil, *recursiveFlag, 0, meta.Modified)
```

Comment:
I'm not sure why it's not working with `meta.Modified|meta.Size`. I will debug it.

Reply:
I think the issue is in the Stream Store.

Reply:
Pushed a fix to this PR.

Reply:
Thanks :)